src/pspm_downsample.m

```diff
@@ -1,9 +1,9 @@
-function [sts, data] = pspm_downsample(data, freqratio)
+function [sts, data, newsr] = pspm_downsample(data, sr,sr_down)
 % ● Description
```

Comment:
Please change help text. This function is now used more generally, not just for users without signal processing toolbox.